fcl/dunefd/mergeana/standard_ana_atmos_dune10kt_1x2x6.fcl

```diff
@@ -0,0 +1,3 @@
+#include "standard_ana_dune10kt_atmo.fcl"
+process_name: Mergeana
+services.LArG4Detector: @local::dune10kt_1x2x6_larg4detector
```

Comment:
This should inherit the entire 1x2x6 service block: https://github.com/DUNE/dunecore/blob/develop/dunecore/Utilities/services_dunefd_horizdrift_1x2x6.fcl#L44

What was the motivation for specifically including the LArG4Detector def?

Reply:
My bad, it was because I saw how [standard_ana_dune10kt_1x2x6.fcl]( https://github.com/DUNE/dunesw/blob/develop/fcl/dunefd/mergeana/standard_ana_dune10kt_1x2x6.fcl) was implemented.  
I updated it. Should I also change `standard_ana_dune10kt_1x2x6.fcl`?

Reply:
Hi @hvsouza 
This change is fine in the immediate but there may become issues down the line if we need to include more dedicated services.

I think a complete fix would be something like:

```
services:
{
  @table::services
  @table::dunefd_1x2x6_simulation_services
}
 ```
 
 Could you add something like this into the atmospheric and standard ana 1x2x6 fcls?

Reply:
That is what I added in my previous commit. I also updated the standard ana 1x2x6 now

Reply:
So you did!
I'm not sure what I was looking at...